modules/authorization/src/admin/relations.ts

```diff
@@ -20,7 +20,7 @@ export class RelationHandler {
   registerRoutes(routingManager: RoutingManager) {
     routingManager.route(
       {
-        path: '/relations',
+        path: '/relation',
```

Comment:
This is breaking the API and convention. Make this `/relations/` and set the createMany variant to `/relations/many` instead.